integration/bundle/testdata/default_python/bundle_summary.txt

```diff
@@ -67,11 +68,6 @@
           "metadata_file_path": "/Workspace/Users/[USERNAME]/.bundle/project_name_$UNIQUE_PRJ/dev/state/metadata.json"
         },
         "edit_mode": "UI_LOCKED",
-        "email_notifications": {
```

Comment:
Hm, any ideas why this is removed? It seems to be email_notifications are still in the template and should be part of the output

Reply:
After testing this manually, i see that `email_notifications` is part of the generated .job.yml, but it is not present on the output of `$ databricks bundle summary`.

Reply:
@anton-107 bundle summary returns the combined json state (as used for TF later), so if `email_notifications` is in .job.yml file, it should be present in the job output in summary as well. @denik any ideas why this happens?

Reply:
I am sorry, this is my mistake -`email_notifications` are indeed in the output of `$ databricks bundle summary` when I test it manually. For some reason they are not when run in this automated test.

Reply:
This field varies per user, so it may differ depending on which account you used.

However, to avoid changing unnecessarily, perhaps you can run it with CLOUD_END=aws and DATABRICKS_HOST/TOKEN set to dogfood? This is how it was generated IIRC.

Reply:
I've reverted all changes in this file that are unrelated to the main change